setup.py

```diff
@@ -45,13 +45,16 @@ def get_long_description():
             'coverage',
             'pytest',
             'webtest',
-            'pyscss'
+            'pyscss',
+            'tox',
+            'flake8'
         ],
```

Comment:
flake8 seems not to be necessary, as it gets added by tox.

Reply:
Done